### PR TITLE
Update year numbers to 2019

### PR DESCRIPTION
**Pull request by Adimote**
There were a few 2017's around, this updates them to 2019.

### CODE REVIEW

File: rules/index.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
```
Comment (RealOrangeOne) [inline rules/index.md]: I'd rather we not ship this page updates until we actually ship the rules, so it's obvious we've not pushed them out yet. Also means we don't ship a known-bad link

File: programming/sr/vision/markers.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ Let's break that down:
 `(v0.5)` tells you the version of the marker, it is important the latest version is used; and
 `'ARENA'` is just a human-readable description of what the marker is for.
 
-Details of the types and size of markers used in the SR2017 game can be found in the [rules](/docs/rules).
+Details of the types and size of markers used in the SR2019 game can be found in the [rules](/docs/rules).
 
-You can download all of the markers as a single [ZIP file](/docs/resources/2017/sr-dev-markers-sr2017.zip)
-or individually from the [git repo](https://github.com/srobo/game-markers/tree/master/SR2017/dev).
+You can download all of the markers as a single [ZIP file](/docs/resources/2019/sr-dev-markers-sr2019.zip)
+or individually from the [git repo](https://github.com/srobo/game-markers/tree/master/SR2019/dev).
```
Comment (PeterJCLaw) [inline programming/sr/vision/markers.md]: These links don't work. Maybe better to remove them for now?

File: rules/index.md (hunk 1 above)
Comment (RealOrangeOne) [reply rules/index.md]: If you're happy to be the one to handle this, then i'm ok with that :+1:

**Comment by PeterJCLaw** [comment]
Since this got superceded by #67, I suggest closing this and we'll reintroduce the relevant links with the right year when we ship things.

**Pull request closed without merging**